### DISCUSSION TITLE
Log qso Cloudlog

### DISCUSSION THREAD

**Discussion by sp5smy** [magicbug/Cloudlog#2124] [General]
In gridtracer I have these settings for the log
https://sp5smy.pzk.pl/Cloudlog-master/index.php/api/qso
test result passed
However, it does not log me in the qso log, it has worked OK so far! :)
plis email sp5smy@linux.pl

**Comment by AndreasK79** [comment]
Hi. Did you set station profile id in GridTracker?

**Reply by phl0** [reply]
Does it support station profiles atfter all yet? 😇

**Reply by AndreasK79** [reply]
It does:

![image](https://user-images.githubusercontent.com/6977712/237062700-7f920edb-7c5f-4348-a4dd-8b1245c656dc.png)

**Reply by phl0** [reply]
Then the op should check wheter he has set the correct station profile ID to log to.

**Reply by sp5smy** [reply]
Everything was working fine until reinstalling the system.
My Gridtracer settings
#
[https://sp5smy.pzk.pl/zapas/22.jpg](url)
[https://sp5smy.pzk.pl/zapas/21.jpg](url)
open in a new settings window !!!

**Reply by phl0** [reply]
Seetings seem to be correct. QSOs are sent to station profile with number one. Is that configured in Cloudlog? And do you have a station location active that includes station profile No 1 so that those QSOs are shown?

**Reply by sp5smy** [reply]
How to check??
And do you have an active station location that includes the profile of station #1 so that these QSOs are shown?

**Reply by phl0** [reply]
User menu and then "Station Locations" and "Station Profiles".

**Reply by sp5smy** [reply]
are you writing about this??
[https://sp5smy.pzk.pl/zapas/33.jpg](url)
open in a new settings window !!!

**Reply by phl0** [reply]
Yes that is the active station profile. Is this linked in the active "Station Logbook"?

**Reply by sp5smy** [reply]
Coś jest źle ustawione w Gridtracer
